config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -38,6 +38,9 @@ message UpsertConfigRequest {
   // optional - only required if config applies to a specific context.
   // If empty, specified config is associated with a default context.
   string context = 4;
+
+  // user details
+  UserDetails user_details = 5;
```

Comment:
Who is the user of a config? And assuming you mean the user making this change, this does not belong in the API - it must be coming from the context.

Reply:
It needs to work the same as tenant ID. The caller doesn't read the context and write it into the request, the server reads the context and persists it. This is the request object.

Reply:
Got it, using requestContext to get email id